server/googleActions.js

```diff
@@ -191,11 +194,12 @@ GoogleActions = {
 			'description' : `${brew.description}`,
 			'parents'     : [folderId],
 			'properties'  : {								//AppProperties is not accessible
-				'shareId'  : nanoid(12),
-				'editId'   : nanoid(12),
-				'title'    : brew.title,
-				'views'    : '0',
-				'renderer' : brew.renderer || 'legacy'
+				'shareId'   : nanoid(12),
+				'editId'    : nanoid(12),
+				'title'     : brew.title,
+				'views'     : '0',
+				'pageCount' : (brew.text.match(/\\page/g) || []).length + 1,
```

Comment:
Same comments as above. Seeing this appear in so may places makes me think that we should indeed have it calculated ahead of time so we don't have to do that here.